### PR TITLE
ENH: Improve kaczmarz example

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: examples/solvers/kaczmarz_tomography.py [hunk 1]
```diff
@@ -30,18 +30,25 @@
...
-m = 1
-ns = 360 // n
-ms = 360 // m
-
-# Create the forward operators.
-ray_trafos = [odl.tomo.RayTransform(space, geometry[i * ns:(i + 1) * ns,
-                                                    j * ms:(j + 1) * ms])
-              for i in range(n) for j in range(m)]
+
+split = 'block'
```
Comment (kohr-h) [inline examples/solvers/kaczmarz_tomography.py]: Add a blank line, so it's better visible as a user parameter.

File: examples/solvers/kaczmarz_tomography.py [hunk 2]
```diff
@@ -30,18 +30,25 @@
...
+if split == 'block':
+    # Split the data into blocks:
+    # 111 222 333
+    n = 20
+    ns = geometry.angles.size // n
+
+    ray_trafos = [odl.tomo.RayTransform(space, geometry[i * ns:(i + 1) * ns])
+                  for i in range(n)]
+elif split == 'slice':
```
Comment (kohr-h) [inline examples/solvers/kaczmarz_tomography.py]: Call this "interlaced" maybe?

Comment (adler-j) [reply examples/solvers/kaczmarz_tomography.py]: Yeah, thats better.

**Comment by adler-j** [comment]
Merge after CI

**Pull request merged**